chapters/angulardart/index.md

```diff
@@ -257,7 +254,8 @@ Connectors are the pieces of code of the architecture that link the elements, or
 #### External entities
 AngularDart has "other systems, software programs, hardware devices or other entities with which the system interacts" (Rozanski and Woods, p. 272), or external entities. These can be extracted from the Context view image below.
 
-![Context view with logos](images/Context_view_with_logos.png "Context view with logos")
+![](images/Context_view_with_logos.png)
+_Context view with logos_
```

Comment:
You are right.

The tricky thing is that to get pandoc with automated numbering and cross referencing right, having _just_ the alt text is not enough. So this leads to some not-so-nice duplication of captions.

I've updated the captions (with a simple sublime edit macro) -- will push the commit shortly.